app/components/Theme.ts

```diff
@@ -0,0 +1,11 @@
+export interface ThemeProps {
```

Comment:
@johnschoeman this almost certainly is an odd place for these types to live. Let me know if you have a recommendation as to where they might go

Reply:
made a comment, but this a bit of a weird case as this really should be defined in the library.

I would duplicate the type in this case and define them in each component that uses it and not export it.  the reasoning being is this isnt an application concept to our app but a type we are putting in to over a third party library, we want to keep this at the boundary of our system and not encourage its use as we dont actually have control over that library.

another option would be to put in the declarations file.

And if they are not stri